debian/autobake-deb.sh

```diff
@@ -58,8 +58,8 @@ remove_rocksdb_tools()
 replace_uring_with_aio()
 {
   sed 's/liburing-dev/libaio-dev/g' -i debian/control
-  sed -e '/-DIGNORE_AIO_CHECK=YES/d' \
-      -e '/-DWITH_URING=YES/d' -i debian/rules
+  sed -e '/-DIGNORE_AIO_CHECK=ON/d' \
+      -e '/-DWITH_URING=ON/d' -i debian/rules
 }
 
 disable_pmem()
```

Comment:
so below needs to be =ON too, or alternately we're waiting for a #2597 merge up.

Reply:
Yes this was the thing that I was thinking back and forward and thinked that waiting #2597 but after thinking it while how long it takes and do we get to same point that PR fixed if we don't do it here.